pagopa/io-app
# feat: [IIP-83] Prevent user from disabling the warning when opening a PN message

## Short description
This PR (temporarily) removes the "Don't show me again" button from the bottom sheet that is presented to the user when opening a message from PN. This has been required by legal.

| Before | After |
| - | - |
| ![Simulator Screen Shot - iPhone 13 Pro - 2022-07-20 at 09 52 04](https://user-images.githubusercontent.com/467098/179928195-cc7a880f-039d-4f2a-8b36-21bd04def994.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2022-07-20 at 09 51 40](https://user-images.githubusercontent.com/467098/179928231-d686af60-3864-4d59-bebf-b0cbf17242c6.png) |

## How to test
Use the [IIP-18-add-pn-endpoints](https://github.com/pagopa/io-dev-api-server/tree/IIP-18-add-pn-endpoints) branch of the dev server and add this to your `config.json`:
```
{
  "messages": {
    "pnCount": 1
  },
  "services": {
    "includePn": true
  }
}
```
Set `PN_ENABLED=YES` to `.env` and run the app, then open a legal message and check that the bottom sheet is always shown. There shouldn't be a way to disable it.
